src/main/scala/com/gu/identity/play/IdUser.scala

```diff
@@ -3,7 +3,8 @@ package com.gu.identity.play
 import play.api.libs.json.Json
 
 case class StatusFields(receiveGnmMarketing: Option[Boolean] = None,
-                        receive3rdPartyMarketing: Option[Boolean] = None)
+                        receive3rdPartyMarketing: Option[Boolean] = None,
+                        userEmailValidated: Option[Boolean] = None)
```

Comment:
personally I'd rejig this to 2 space indent with the brackets in the usual place but I can see it was already indented that way before
```scala
case class StatusFields(
  receiveGnmMarketing: Option[Boolean] = None,
  receive3rdPartyMarketing: Option[Boolean] = None,
  userEmailValidated: Option[Boolean] = None
)
```